dataland-keycloak/realms/datalandsecurity-realm.json

```diff
@@ -1514,7 +1527,19 @@
       }
     }
   ],
-  "identityProviderMappers": [],
+  "identityProviderMappers": [
+    {
+      "id": "8fe5cb09-9a63-4a4a-aa51-32d78aafc187",
+      "name": "picture",
+      "identityProviderAlias": "google",
+      "identityProviderMapper": "google-user-attribute-mapper",
+      "config": {
+        "syncMode": "FORCE",
+        "jsonField": "picture",
+        "userAttribute": "picture"
+      }
+    }
+  ],
   "components": {
     "org.keycloak.services.clientregistration.policy.ClientRegistrationPolicy": [
       {
```

Comment:
Ich bin mir nicht sicher, ob das mit dem Branch hier zu tun hat, aber der Social Login über Google geht bei mir nicht mehr.
Ich werde nicht auf diese Google Maske weitergeleitet.

Reply:
Jetzt geht der Social Login.
Aber was mir aufgefallen ist:
Ich habe kein Profile Pic bei meinem Google Konto. Dadurch wird Folgendes angezeigt:

![image](https://user-images.githubusercontent.com/99255296/177984249-d9dfdaf6-b2c0-4e1f-b1df-3292fdca1d84.png)


Sollen wir in dem Fall nicht vllt etwas anderes nehmen?   Z.B. erstmal auch das Dataland Logo wie bei einem "normalen" Login?

Reply:
Hmm. Das ist in der tat komisch. Normalerweise sollte bei keinem profile pic ein Fallback-Bild angezeigt werden. Das werde ich nochmal überprüfen